### PR TITLE
Feature/us18318 mass edit donations

### PR DESCRIPTION
**Pull request by vchtelmakh**
- Any adjustments with an equal debit/credit to an account should be removed. E.g. $50 debit/$50 credit to same account is a wash and the "adjustment" should not be sent, as nothing is changed.
- If an adjusting entry has both a debit and a credit, it should be netted. E.g. $30 debit, $40 credit, should be a $0 debit and $10 credit - we only care about the net value

### CODE REVIEW

File: Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+
+        public MpJournalEntry CreateNewJournalEntry(string batchId, MpDistributionAdjustment mpDistributionAdjustment)
+        {
+            var mpJournalEntry = new MpJournalEntry
+            {
+                BatchID = batchId,
+                CreatedDate = DateTime.Now,
+                ExportedDate = null,
+                Description = "test desc", // TODO: understand what goes here
```
Comment (briancavanaugh) [inline Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs]: Going to update this?

File: Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            }
+
+            return mpJournalEntry;
+        }
+
+        public MpJournalEntry NetCreditsAndDebits(MpJournalEntry journalEntry)
+        {
+            if ( IsNetCredit(journalEntry) )
+            {
+                journalEntry.CreditAmount -= journalEntry.DebitAmount;
```
Comment (briancavanaugh) [inline Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs]: Does Math.Abs need to be applied here to the DebitAmount like earlier in this class?

Comment (vchtelmakh) [reply Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs]: No, but good catch, had to think about it for a sec. `MpDistributionAdjustment` has an `amount` property which is positive or negative.  The negative or positive determines whether it goes into a debit or credit property in `MpJournalEntry`. But by the time it's in a `MpJournalEntry`, it should already be a positive number.

File: Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs (hunk 1 above)
Comment (vchtelmakh) [reply Crossroads.Service.Finance/Services/JournalEntry/JournalEntryService.cs]: Not yet, Amy hasn't decided what she wants here.

**Pull request merged**